ledgers/xero/lib/ledger_sync/ledgers/xero/contact/serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Ledgers
+    module Xero
+      class Contact
+        class Serializer < Xero::Serializer
```

Comment:
Now that we are making names match and resources are ledger-specific, these definitions seem redundant. We could just infer them from the resource attributes. The only special piece here is the `ledger_id` and if we use any types.